### ISSUE TITLE
Regular Crash / Freeze

### ISSUE THREAD

**Issue by vedicium** [balena-os/balena-up-board#60]
This issue is regarding [this topic](https://forums.balena.io/t/regular-crashes/4501) on the Balena Forum.

**TL;DR**
OS Crashes after an hour or 2, already added `i915.enable_rc6=0` for the screen flickering/glitching. Still crashes/freezes after a few hours. Tested with balenaOS 2.26.0+rev3 for the up-board, running on an UP Squared Intel Atom E3940

**Extra Information**
Board: UP Squared Intel Atom E3940
Connection: Ethernet
Display Connection: DisplayPort
BalenaOS Version: balenaOS 2.26.0+rev3
BalenaOS Supervisor: 8.0.0
Running on balenaCloud

<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_fncr)

**Comment by spanceac** [comment]
This was fixed by https://github.com/balena-os/balena-up-board/pull/62/commits/1af9b1a14cabbffa9f29bba11e0a3883cf319791
